tests/test_dcos_e2e/test_cluster.py

```diff
@@ -136,6 +138,52 @@ def test_custom(self, cluster_backend: ClusterBackend) -> None:
             assert len(cluster.public_agents) == public_agents
 
 
+class TestCopyFiles:
```

Comment:
I guess this needs to go in `.travis.yml` and `admin/run_script.py`.

Reply:
It did, I didn't push it :)